From 6057d8dc3be5aef007e7c98cf8bb7a1d8369e76d Mon Sep 17 00:00:00 2001
From: xan <xantares09@hotmail.com>
Date: Sun, 14 Aug 2016 12:34:36 +0200
Subject: [PATCH 1/2] Fix MSVC Python compilation detection

PYTHON_WORKING_DIR must be the same binary location as specified for
try_compile.
---
 src/Python/CMakeLists.txt | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/Python/CMakeLists.txt b/src/Python/CMakeLists.txt
index e622ab8..0ada2ac 100644
--- src/Python/CMakeLists.txt
+++ src/Python/CMakeLists.txt
@@ -59,11 +59,7 @@ IF(OPENMESH_BUILD_PYTHON_BINDINGS)
 
 				IF(PYTHONINTERP_FOUND)
 					
-					IF(MSVC)
-						SET(PYTHON_WORKING_DIR "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_TRY_COMPILE_CONFIGURATION}")
-					ELSE()
-						SET(PYTHON_WORKING_DIR "${CMAKE_CURRENT_BINARY_DIR}")
-					ENDIF()
+					set (PYTHON_WORKING_DIR ${CMAKE_CURRENT_BINARY_DIR})
 					
 					EXECUTE_PROCESS(
 						COMMAND ${PYTHON_EXECUTABLE} -c "from example import *; greet(); planet = World()"
-- 
2.9.2

